src/service.py

```diff
@@ -10,15 +10,16 @@
 import pandas as pd
 import urllib3
 
-
 class FastTextServiceBase(mlservicewrapper.core.services.Service):
 
     async def load(self, ctx: mlservicewrapper.core.contexts.ServiceContext):
         model_path = self.get_model_path(ctx)
-        
+
         if not os.path.exists(model_path):
+            model_parent_path = model_path.rsplit(os.sep, 1)[0]
+            if not os.path.exists(model_parent_path):
+                os.makedirs(model_parent_path)
```

Comment:
And same here, functionality is identical, but the [`exist_ok`](https://docs.python.org/3/library/os.html#os.makedirs) parameter will eliminate the need for the extra `if` block.